catalog/data_catalogs.go

```diff
@@ -0,0 +1,25 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceCatalogs() *schema.Resource {
+	var data struct {
+		Ids []string `json:"ids,omitempty" tf:"computed,slice_set"`
+	}
+	return common.DataResource(&data, func(ctx context.Context, c *common.DatabricksClient) error {
+		catalogsAPI := NewCatalogsAPI(ctx, c)
+		catalogs, err := catalogsAPI.list()
```

Comment:
How do we select which metastore to query? Or catalog names should be unique across all metastores? I don’t remember

Reply:
There's no way to select metastore